Cargo.toml

```diff
@@ -2,8 +2,10 @@
 members = [
   "fake_data_generator",
   "api",
-  "ingestion"
+  "ingestion",
+  "lard_tests",
```

Comment:
I would remove lard from the package name and path, to match consistency with the rest of the workspace. Also maybe "integration_tests" would be a better name anyway?